server/schedulers/hot_region.go

```diff
@@ -248,6 +248,14 @@ func (h *hotScheduler) summaryPendingInfluence() {
 			to.AddInfluence(&p.origin, weight)
 		}
 	}
+	for storeID, info := range h.stInfos {
```

Comment:
How to clear the stale metrics? like the store is removing or gone.

Reply:
https://github.com/tikv/pd/blob/e74488f8b121adb751757d071fe3b71993c7b5b9/server/cluster/coordinator.go#L606